src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,26 @@
+const jwtService = require('../services/jwt.service.js');
+
+const authMiddleware = (req, res) => {
```

Comment:
The `authMiddleware` function should accept a third parameter `next` to properly function as middleware in Express. Update the function signature to `const authMiddleware = (req, res, next) => {`.